samples/snippets/fsharp/azure/blob-storage.fsx

```diff
@@ -5,31 +5,28 @@ open Azure.Storage.Blobs.Models
 open Azure.Storage.Blobs.Specialized
 open System.Text
 
+System.Environment.CurrentDirectory <- __SOURCE_DIRECTORY__
+
 //
 // Get your connection string.
 //
 
 let storageConnString = "..." // fill this in from your storage account
-(*
-// To fetch the connection string from a configuration file.
-let storageConnString = 
-    CloudConfigurationManager.GetSetting("StorageConnectionString")
-*)
 
 //
 // Create some local dummy data.
 //
 
 // Create a dummy file to upload
-let localFile = __SOURCE_DIRECTORY__ + "/myfile.txt"
+let localFile = "/myfile.txt"
```

Comment:
```suggestion
let localFile = "./myfile.txt"
```

Depending on the host environment, the path in its existing form could be interpreted as relative or starting at the root of the filesystem.   This form ensures that it is interpreted as relative.